### ISSUE TITLE
Feature request: better music system

### ISSUE THREAD

**Issue by Doomstar00** [yairm210/Unciv#12099]
### Before creating

- [X] This is NOT a gameplay feature from Civ VI, BNW, or outside - see Roadmap
- [X] This is NOT a gameplay feature from Vanilla Civ V or from G&K - If so, it should be a comment in https://github.com/yairm210/Unciv/issues/4697

### Problem Description

It is annoying, when in early singleplayer game(when turns short) or when i check diplomacy or when i mass destroy city states, manually selected music replaced with trigger music, and i have to select manually after each trigger event

### Desired Solution

change music system:
1) Peace/War music should not play every 10 turns, it should just have higher priority when game chooses next track, also peace/war without civ prefix should be used for all civs (i suggest 50% for civ - peace/war, 25% for peace/war without civ and 25% for ambient);
2) Theme should not be played on victory screen because of existence of Victory suffix(and especially civname - musicname - theme when i do not play as civname)
3) for all notifications(first contact, war declaration on player, defeat of enemy,  ...) and menus(diplomacy, victory screen when smb. won, ...) game should pause main background track and turn on event music, but stop event music and continue playing main background track when menu closed;
4) i also suggest to pause main music when player select civ in new game screen or in map editor, play civ music for 10-20 seconds and then stop it and play main(because there player cannot close notification)

EXAMPLE: 
I launch Unciv, gsme start playing(as now) any Ambient, i create new game and select Russia as my civ, game pause main and play Russia - thene/peace(/war?) for 20 seconds then stop(or stop if i start game earlier), as next background track game select Russia - peace(50%), peace(25%) or amvient(25%), i find America(America - theme until i close notification, than bacground(russia - peace gor example)), i open diplomacy menu and select America(america - peace), declare war(america - war), close menu(russia - peace because it has not ended), game choose next background track(50% russia - war, 25% any - war, 25% ambient), i destroy America civ, game (as i know) opens victory screen(russia - victory or any - victory, but NOT Rapture - Theme(from "Rapture from Bioshock in Unciv" mod if installed)), i close victory screen and see notification about America's defeat(america - defeat or any - defeat), i close notification(continue russia - war)

### Alternative Approaches

-

**Comment by Ouaz** [comment]
>  It is annoying, when in early singleplayer game, when i check diplomacy, manually selected music replaced with trigger music

Actually, the civ music wasn't played when entering the Diplomacy screen before.

I agree it is annoying, considering that the Diplomacy screen can be checked numerous times in a game session.

**Comment by Doomstar00** [comment]
it plays, when you sekect civ(otherciv - Peace or otherciv - War)

**Comment by Doomstar00** [comment]
will this be added?

or refused?

**Comment by RobLoach** [comment]
Have patience. Everyone has their own priorities.

**Comment by Doomstar00** [comment]
.

**Comment by Doomstar00** [comment]
.

**Comment by Ouaz** [comment]
Aw, I didn't manage to find this topic in my notifications history, and now it is closed , it popped up. ^^

Anyway, I wanted to say this request was legitimate.

Currently, an ambient music is cut (and plays a dedicated civ theme -peace or war- instead) when:

1) the player meets a civ for the first time = Good (play the peace theme of the encountered civ)
2) a civ declares war on the player = Good (plays the war theme of the civ declaring war)
3) every 10th turn = Okayish (it plays the player civ's peace/war theme each 10 turns)

4) each time a civ is selected in the diplomacy screen = **BAD**

It cuts any played tracks to start the selected civ theme. So let's say one simply wants to check a thing for 2 seconds in the diplomacy screen, and the same theme starts and is played again and again. Or cycle through all civs in the diplomacy screen and the music changes and plays the same starting notes again and again, then the whole music of the last selected civ when leaving the diplomacy screen . This makes no sense. ^^

This wasn't the case (AFAIK) when the custom music tracks feature were added by SomeTroglodyte.

So basically, for this one, the suffix should be changed to `(Diplomacy)` for example.
So the main music is not cut each time the player enters the diplomacy screen (if he does not provide a `CivName - Diplomacy` track)

Description | Prefix | Suffix
-- | -- | --  
Diplomacy: Select player | (nation name) | Peace or War | 
to |  |
Diplomacy: Select player | (nation name) | Diplomacy | 

https://github.com/yairm210/Unciv/blob/master/docs/Modders/Images-and-Audio.md#context-sensitive-music-list-of-triggers
